packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -138,13 +138,17 @@ class InnerBlocks extends Component {
 			clientId,
 			hasOverlay,
 			__experimentalCaptureToolbars: captureToolbars,
+			onInput,
+			isSelected,
 			...props
 		} = this.props;
 		const { templateInProcess } = this.state;
 
 		const classes = classnames( 'block-editor-inner-blocks', {
 			'has-overlay': enableClickThrough && hasOverlay,
 			'is-capturing-toolbar': captureToolbars,
+			'is-controlled': onInput,
+			'is-selected': isSelected,
```

Comment:
Could we add these to `BlockList`? I'm trying to move the existing classes in #19849.

Reply:
That's because this particular wrapper will eventually be opt-out: #19910.

Reply:
Yeah, sure.

Reply:
I've moved these classes to block-list-block now. Does that change make sense??